### ISSUE TITLE
Perl6 REPL can't find liblinenoise.so

### ISSUE THREAD

**Issue by vytas-dauksa** [hoelzro/p6-linenoise#9]
Perl6 REPL doesn't start after installing Linenoise

```
vytas-local@vytas-desktop:/tmp$ panda --force install Linenoise
==> Fetching Linenoise
==> Building Linenoise
gcc -c -fPIC -Wdeclaration-after-statement -Werror=declaration-after-statement -O3 -DNDEBUG  -D_REENTRANT -D_FILE_OFFSET_BITS=64 -fPIC -o linenoise.o linenoise.c
gcc -shared -fPIC  -O3 -DNDEBUG -Wl,-rpath,/home/vytas-local/.rakudobrew/moar-nom/install/lib -Wl,-rpath,/home/vytas-local/.rakudobrew/moar-nom/install/share/perl6/site/lib -lm -lpthread -lrt -ldl -o /tmp/.panda-work/1449240020_1/blib/lib/liblinenoise.so linenoise.o
gcc -o constant-helper -Wdeclaration-after-statement -Werror=declaration-after-statement -O3 -DNDEBUG  -D_REENTRANT -D_FILE_OFFSET_BITS=64 -fPIC constant-helper.c
perl6 fill-constants.pl < lib/Linenoise.pm.in > lib/Linenoise.pm
==> Testing Linenoise
==> Installing Linenoise
==> Successfully installed Linenoise
vytas-local@vytas-desktop:/tmp$ panda ^C
vytas-local@vytas-desktop:/tmp$ perl6
I ran into a problem while trying to set up REPL completions: Cannot locate native library 'liblinenoise.so': liblinenoise.so: cannot open shared object file: No such file or directory
Continuing without tab completions

===SORRY!===
Cannot locate native library 'liblinenoise.so': liblinenoise.so: cannot open shared object file: No such file or directory

vytas-local@vytas-desktop:/tmp$ perl6 -v
This is rakudo version 2015.11-353-g734549a built on MoarVM version 2015.11-34-gc3eea17 implementing Perl v6.b.

```

**Comment by hoelzro** [comment]
@vytas-dauksa Thanks for the report; this is due to a recent change in how module installation works with Rakudo.  I'll fix this up if there's a change needed on the module side after the necessary code lands in Rakudo.

**Comment by hoelzro** [comment]
I got this working through a horrible hack (9f79f92) and have created an issue to use `%?RESOURCES` after it's been implemented (#10)
